packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -201,7 +205,7 @@ function ListView<T extends object>(props: ListViewProps<T>, ref: DOMRef<HTMLDiv
   }
 
   return (
-    <ListViewContext.Provider value={{state, keyboardDelegate, dragState, onAction, isListDraggable, layout}}>
+    <ListViewContext.Provider value={{state, keyboardDelegate, dragState, onAction, isListDraggable, layout, overflowMode}}>
```

Comment:
could we do the overflow mode in one place like Table does? https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/table/src/TableView.tsx#L302
or did that not work?

Reply:
I tried that at first, but the grid and draggable stuff got between the 'wrap' class and the item class. I also saw a conditional class being assigned so was worried that would also affect it. And I didn't want to have a super long css class identifier.